uint8 robotID
string robotColor
float32 x
float32 y
float32 angle